### ISSUE TITLE
Canon 100D - No resolution controls?

### ISSUE THREAD

**Issue by nekohayo** [Atterratio/obs-gphoto#11]
Trying out this plugin by installing it from git (as per your instructions) on Fedora 30's version of OBS, and quite impressed that it manages to see and composite the camera through gphoto2, which is more than anything else I've seen outside the Linux world...

There's a a couple of problems though: it crashes if I toggle the source on and off in OBS (I'll file another ticket about that) and it doesn't let me specify the resolution in the settings, so the image always shows up as something that looks like 4:3 or 3:2 aspect ratio, instead of the 1920x1080 that is the camera's native video recording resolution. Is it supposed to be able to let me specify a capture resolution?

**Comment by mjg** [comment]
No, this uses the live preview images over USB which are not at FullHD, let alone the full sensor resolution. It's the same with gphoto directly or Canon's new Windows tool.

With an 80D I get 960x640 in photo mode and 1024x576 in video mode, which is to be expected.

For FullHD you would need to capture the HDMI output which - depending on your model and mode - is not "clean" (shows controls).
